### PR TITLE
Fix email rendering and on_after_reset_password

### PR DESCRIPTION
**Pull request by edulauer**
- Fix HTML rendering for email template
- Split email configuration into new file
- Add route to function on_after_reset_password

### CODE REVIEW

**Comment by washolive** [comment]
I understand that "html=body" has fixed the html format problem when sending email, right? 
Good idea to move email configuration to other file. 
Thanks!

**Review by washolive — APPROVED** [review]
I understand that "html=body" has fixed the html format problem when sending email, right?
Good idea to move email configuration to other file.
Thanks!

File: api.py [hunk 1]
```diff
@@ -36,28 +36,17 @@
 
 oauth2_scheme = OAuth2PasswordBearer(tokenUrl="auth/jwt/login")
 
+@app.post('/email')
```
Comment (augusto-herrmann) [inline api.py:39]: Is the idea to expose the ability to send email in the API for users to use? Why? Can't this be abused for SPAM?

File: api.py [hunk 2]
```diff
@@ -75,23 +64,23 @@ async def on_after_forgot_password(user: UserDB, token: str, request: Request):
...
+            Segue seu novo token de acesso: <br/> {token}
             </p>
             </body>
             </html>
             """
     await send_email([user.email], subject, body)
 
 async def on_after_reset_password(user: UserDB, request: Request):
-    logging.info("User %s has reset their password.", user.id)
+    print("User %s has reset their password.", user.id)
```
Comment (augusto-herrmann) [inline api.py:77]: Does `print` work better than `logging.info` here? Why?

**Review by augusto-herrmann — CHANGES_REQUESTED** [review]
Were no changes made in the tests for this?

**Comment by edulauer** [comment]
> stand that "html=body" has fixed the html format problem when sending email, right?
> Good idea to move email configuration to other file.
> Thanks!

Yes, exactly

File: api.py (hunk 2 above)
Comment (edulauer) [reply api.py:77]: Sorry...actually I was testing the print function but logging is better, and I forgot to revert this change.

**Comment by edulauer** [comment]
> Were no changes made in the tests for this?

No. In this pull request we didn't predict tests for email sending. For sure it's a good idea to implement it.

File: api.py (hunk 1 above)
Comment (edulauer) [reply api.py:39]: You're right, I think we can suppress this line...

Comment (augusto-herrmann) [inline api.py:39]:
```suggestion
```

Removing the line as discussed above.

File: api.py (hunk 2 above)
Comment (augusto-herrmann) [reply api.py:77]:
```suggestion
    logging.info("User %s has reset their password.", user.id)
```
Ok, so let's revert this line.

**Review by augusto-herrmann — CHANGES_REQUESTED** [review]
Ok, I see now that there were no tests for email sending before. Would you rather implement the tests now, or merge without it and add it later, @edulauer?

**Review by augusto-herrmann — APPROVED** [review]
I'm ok with doing it later. It was not being tested before, anyway.

**Comment by edulauer** [comment]
> I'm ok with doing it later. It was not being tested before, anyway.

Ok, agree

**Pull request merged**